eslint.config.js

```diff
@@ -0,0 +1,15 @@
+import eslintConfigPrettier from 'eslint-config-prettier'
+import js from '@eslint/js'
+import globals from 'globals'
+
+export default [
+  js.configs.recommended,
+  eslintConfigPrettier,
+  {
+    languageOptions: {
+      globals: {
+        ...globals.browser,
+      },
```

Comment:
Eslint complaining about browser globals not existing, like the `window`

<img width="964" alt="CleanShot 2024-05-22 at 06 30 30@2x" src="https://github.com/vigetlabs/craft-site-starter/assets/2145998/5aa87ecc-1e5b-4f34-b57a-881f6faa2ef7">